pkg/types/appsec_event.go

```diff
@@ -187,11 +184,8 @@ func (w MatchedRules) ByTags(match []string) MatchedRules {
 func (w MatchedRules) ByTag(match string) MatchedRules {
 	ret := MatchedRules{}
 	for _, rule := range w {
-		for _, tag := range rule["tags"].([]string) {
-			if tag == match {
-				ret = append(ret, rule)
-				break
-			}
+		if slices.Contains(rule["tags"].([]string), match) {
+			ret = append(ret, rule)
```

Comment:
Missing the break inside the if statement

Reply:
yes, there's no loop anymore to break